sklearn/preprocessing/tests/test_data.py

```diff
@@ -2227,6 +2227,15 @@ def test_categorical_encoder_dtypes_pandas():
     assert_array_equal(enc.transform(X).toarray(), exp)
 
 
+def test_categorical_encoder_warning():
+    # check no warning is raised
```

Comment:
"check no" -> "check that no" in the comment. Also maybe move this comment above the `assert` statement below...